apps/api/src/app/testing/translations/delete-translation-group.e2e-ee.ts

```diff
@@ -0,0 +1,37 @@
+import { UserSession } from '@novu/testing';
+import { expect } from 'chai';
+
+describe('Delete a Transaltion group - /translations/group/:id (Delete)', async () => {
+  let session: UserSession;
+
+  beforeEach(async () => {
+    session = new UserSession();
+    await session.initialize();
+
+    await session.testAgent.put(`/v1/organizations/language`).send({
+      locale: 'hi_IN',
+    });
+  });
+
+  it('should delete the translation group', async function () {
```

Comment:
Maybe we should add a test that also checks that the group's translations were also deleted

Reply:
Done :)